cardano_clusterlib/clusterlib.py

```diff
@@ -1692,6 +1692,46 @@ def get_kes_period(self) -> int:
         """Return last block KES period."""
         return int(self.get_slot_no() // self.slots_per_kes_period)
 
+    def get_kes_period_info(
+        self, issue_op_cert: FileType, operational_certificate_issue_counter: FileType
+    ) -> Dict:
```

Comment:
This can be simply `dict` when we don't want to use the generic version with additional type information (like `Dict[str, Union[list, dict]]`).